Automattic/woocommerce-payments
# Modify text of onboarding success card and make it dismissible when fully verified

Fixes #5965

#### Changes proposed in this Pull Request

- Update the copy for the success messages after first part of PO and after the second part, and the full KYC verification.
- Make the second card, after the whole PO has completed and after full KYC, dismissable.



<table>

<tr><th>After PO</th><td><img width="723" alt="Screenshot 2023-04-07 at 15 02 32" src="https://user-images.githubusercontent.com/2612426/230573290-8bbe7fb7-0556-4286-90e3-df38e0d29221.png"></td>
<tr><th>After Full KYC</th><td><img width="723" alt="Screenshot 2023-04-07 at 14 58 35" src="https://user-images.githubusercontent.com/2612426/230573300-29a77b68-0088-4aec-8812-ab07cc15ff04.png"></td>
</table>



#### Testing instructions

- Go to Payments → Overview and add &wcpay-connection-success=1 to the URL.
- The After Full KYC notice should be visible. This will have 3 dots in the right corner, with a dropdown menu and a "Dismiss" action to hide the card.
- Edit [get_account_status_data](https://github.com/Automattic/woocommerce-payments/blob/f61304b494a35eb7d394030fb3a43fc8d3c0118d/includes/class-wc-payments-account.php#L228) to return isEnabled => true and isComplete => false, and refresh.
- The After PO notice should be visible. This will not be dismissable.

You can also test it by onboarding a new account, but this PR only updates the UI, so no for the logic to be tested.

-------------------


**Post merge**
